app/uploaders/exported_photos.rb

```diff
@@ -9,7 +9,14 @@ def store_dir
     "uploads/users"
   end
 
+  def extension_allowlist
+    %w[zip]
+  end
+
   def filename
-    "#{model.username}_photos_#{secure_token}.zip" if original_filename.present?
+    if original_filename.present?
```

Comment:
Style/GuardClause: Use a guard clause (`return unless original_filename.present?`) instead of wrapping the code inside a conditional expression.